commands/poll.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+	name: 'poll',
+	ping: true,
+	description: 'Creates a poll with the given question and options. Usage: "bb poll Question? Option1 Option2 Option3"',
+	permission: 100,
+	category: 'Random command',
+	noBanphrase: true,
+	execute: async (channel, user, input, perm) => {
+		try {
+			if (module.exports.permission > perm) {
+				return;
+			}
+			if (input.length < 4) {
+				return 'Usage: "bb poll Question? Option1 Option2 Option3"';
+			}
+			const question = input.slice(2).join(' ').replace('?', '');
```

Comment:
With this, the question will also include the answers

Reply:
[Clueless](chat.vote)